.env.example

```diff
@@ -24,7 +24,7 @@ LOG_LEVEL=debug
 DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
 DB_PORT=3306
-DB_DATABASE=laramerce
```

Comment:
Bring this back. Don't modify the example env unless you've asked the maintainers.